### PR TITLE
Awssteph multi payer

### PR DESCRIPTION
**Pull request by awssteph**
Updated both yaml and py with multi payer look through. Changed some of the order of the python to make this work so please test on two payers.

### CODE REVIEW

File: create_rls.py [hunk 1]
```diff
@@ -12,33 +11,27 @@
     "Missing bucket for uploading CSV. Please define bucket as ENV VAR BUCKET_NAME")
 TMP_RLS_FILE = os_environ['TMP_RLS_FILE'] if 'TMP_RLS_FILE' in os_environ else '/tmp/cudos_rls.csv'
 RLS_HEADER = ['UserName', 'account_id']
-ROOT_OU = os_environ['ROOT_OU'] if 'ROOT_OU' in os_environ else exit("Missing ROOT_OU env var, please define ROOT_OU in ENV vars")
+#ROOT_OU = os_environ['ROOT_OU'] if 'ROOT_OU' in os_environ else exit("Missing ROOT_OU env var, please define ROOT_OU in ENV vars")
```
Comment (vmindru) [inline create_rls.py]: let's delete this one

File: create_rls.py [hunk 2]
```diff
@@ -12,33 +11,27 @@
...
 ACCOUNT_ID = boto3.client('sts').get_caller_identity().get('Account')
-QS_REGION = 'eu-central-1'
+QS_REGION = os_environ['QS_REGION']
```
Comment (vmindru) [inline create_rls.py:17]: add
MANAGEMENT_ACCOUNT_IDS = os_environ['MANAGEMENT_ACCOUNT_IDS'] if 'MANAGEMENT_ACCOUNT_IDS' in os_environ else ACCOUNT_ID

File: create_rls.py [hunk 3]
```diff
@@ -144,27 +137,36 @@ def upload_to_s3(file, s3_file):
...
+        qs_users = get_qs_users(ACCOUNT_ID, qs_client)
+        qs_users = {qs_user['UserName']: qs_user['Email'] for qs_user in qs_users}
+        qs_email_user_map = {}
+        for key, value in qs_users.items():
+            if value not in qs_email_user_map:
+                qs_email_user_map[value] = [key]
+            else:
+                qs_email_user_map[value].append(key)
+        qs_rls = {}
+        for entry in ou_tag_data:
```
Comment (vmindru) [inline create_rls.py:160]: I had a bug here, replace this for cycle to 

        if entry in qs_email_user_map:
            for qs_user in qs_email_user_map[entry]:
                if qs_user not in qs_rls:
                    qs_rls[qs_user] = ou_tag_data[entry]
                else:
                    qs_rls[qs_user] = ou_tag_data[entry.extend(entry)]

File: create_rls.py (hunk 1 above)
Comment (awssteph) [reply create_rls.py]: Done

File: create_rls.py (hunk 2 above)
Comment (awssteph) [reply create_rls.py:17]: Done

File: create_rls.py (hunk 3 above)
Comment (awssteph) [reply create_rls.py:160]: When i add this in its not working, can you pull down my update and add it and push to the branch please

Comment (awssteph) [reply create_rls.py:160]: Done

**Pull request closed without merging**